design-patterns/gc-intro/_base.scss

```diff
@@ -0,0 +1,5 @@
+/* REMOVE ONCE IMPLEMENTED IN WET-BOEW */
```

Comment:
This can also be implemented as an utilities in GCWeb. If that option it's chosen, then that new feature should go in the common component list under "colour" along with the other background CSS selector.

See: http://localhost:4000/common/colour/colour-en.html

Reply:
Not needed anymore.